src/components/MachineSelector/tests/mockData.ts

```diff
@@ -0,0 +1,3 @@
+export const fakeMachines = {
```

Comment:
fakeMachines -> mockMachineURLs

Reply:
I dont think we should ad URLs athe end as it can be simple IP or names of machines what do you say

Reply:
CC: @Harkishen-Singh

Reply:
I agree with Aquib. Test data should prefer the mock... convention.

Reply:
I mean we can call it mockMachines not mockMachineURLS